runner/runners/facade.go

```diff
@@ -0,0 +1,13 @@
+package runners
+
+import (
+	"github.com/scootdev/scoot/runner"
+)
+
+// ServiceFacade makes a Service from component parts.
```

Comment:
Why not just RunnerService? Facade implies testing usage to me (as does file name)